docs/developers/HowToRelease.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: How to Release
+nav_order: 17
+parent: Getting-Started
```

Comment:
Please update accordingly. Thanks.

Reply:
> It seems there are quite some issues with the current nav orders.

Unfortunately, after a quick check, I think I need more time for doing this.

I've update the PR to make the new page correctly show up, so we can merge this first.